src/components/Dashboard/Header/Header.jsx

```diff
@@ -4,19 +4,43 @@ import Nav from 'react-bootstrap/Nav';
 import { Link } from 'react-router-dom';
 import avatar from '../../assets/avatar.svg';
 import './Header.scss';
+import Logout from "./Authentication/Logout";
+
 
 export default class NavBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoggedIn: props.isLoggedIn,
+    };
+  }
+
   render() {
+    const { isLoggedIn } = this.state;
     return (
       <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
         <Navbar.Brand><Link to="/">Add a thing</Link></Navbar.Brand>
         <Navbar.Toggle aria-controls="responsive-navbar-nav" />
         <Navbar.Collapse id="responsive-navbar-nav" className="justify-content-end">
-          <Nav>
-            <Nav.Link href="#login">
-              {'Log in'}
-            </Nav.Link>
-          </Nav>
+          {isLoggedIn
+            ? (
+              <Nav>
+                <Nav.Link href="/logout">
```

Comment:
Link component from `react-router-dom` uses attribute `to` instead of `href` to seamlessly transition between app states without ever reloading the page.

Reply:
Please, provide an example of how to should be used in this case, because to doesn't work for me.

Reply:
I'm going to do this on the next Sprint in the scope of refactoring.

Reply:
No, this has to be done before this demo. Plenty of time for that